arch/sim/src/Makefile

```diff
@@ -285,14 +285,14 @@ nuttx-names.dat: nuttx-names.in
 nuttx$(EXEEXT): libarch$(LIBEXT) board/libboard$(LIBEXT) $(HEADOBJ) $(LINKOBJS) $(HOSTOBJS) nuttx-names.dat
 	$(Q) echo "LD:  nuttx$(EXEEXT)"
 	$(Q) $(LD) -r $(LDLINKFLAGS) $(RELPATHS) $(EXTRA_LIBPATHS) -o nuttx.rel $(REQUIREDOBJS) $(LDSTARTGROUP) $(RELLIBS) $(EXTRA_LIBS) $(LDENDGROUP)
-ifeq ("$(shell $(CC) --version | grep clang)","")
-   # none clang based native compilers need opjcopy to build simulation
+ifneq ($(CONFIG_HOST_MACOS),y)
+	# none clang based native compilers need opjcopy to build simulation
```

Comment:
it's wrong with the new CONFIG_HOST_MACOS guard.

Reply:
The check isn't related to clang now, we need adjust the comment too.

Reply:
done